Computation Payload Manifest / Outbound Net related changes (#130)
golemfactory/yagna-docs: provider-tutorials/provider-cli.md

**Note that if you decided to use an external wallet during your setup process, you can connect to ZkSync's wallet at** [**https://wallet.zksync.io/**](https://wallet.zksync.io/) **and exit that way. In the scenario that a different payment driver is being used, you will need to use the relevant available options to connect and access your tokens.**

## Advanced Settings

> `ya-provider` allows to fine-tune config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-unit`.

Reviewer: ```suggestion
`ya-provider` allows to fine-tune the config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-unit`.
```

> `ya-provider` allows to fine tune config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-utnit`.

Reviewer 2: ```suggestion
`ya-provider` allows to fine-tune config created with `golemsp settings` using commands `config`, `preset`, `profile`, and `exe-unit`.
```

> Additionally, it allows configuring certificate _keystore_ and domain _whitelist_.

Reviewer: I'm unsure if it's necessary to accent these words ... plus, there's a repetition... and generally, maybe just a bit more context would be due... 

```suggestion
Additionally, it enables configuration of the certificate keystore and of the domain whitelist, which are used to control what kind of outbound traffic is allowed out of the VM containers run by the requestors on your machine.
```

> Additionally it allows to configure certificate _keystore_ and domain _whitelist_.

Reviewer 2: ```suggestion
Additionally, it allows configuring certificate _keystore_ and domain _whitelist_.
```

### Keystore

> Provider has an embedded certificate keystore.

Reviewer 2: ```suggestion
The provider has an embedded certificate keystore.
```

> The provider has an embedded certificate keystore.

Reviewer: I think the first sentence should state the _purpose_ of this keystore... e.g.:

```suggestion
The provider has an embedded certificate keystore which is used to validate any additional permissions for the payload launched by the requestors.
```

By default it contains only Golem public certificate which allows to execute [example app](../requestor-tutorials/service-development/service-example-6-external-api-request.md) and apps from trusted by Golem creators (certificates allow to verify incoming _Demand_'s [Computational Payload Manifests](../requestor-tutorials/vm-runtime/computation-payload-manifest.md)).

Run `ya-provider keystore --help` to see possible subcommands

### Domain whitelist

> _Demand_'s [Computational Payload Manifests](../requestor-tutorials/vm-runtime/computation-payload-manifest.md) can specify list of URLs which may get called by service running on a Provider. If Manifest declares requests to URLs which domains are not whitelisted it must come with a [signature and app author's public certificate](../requestor-tutorials/vm-runtime/computation-payload-manifest.md). By default domains `whitelist` is empty.

Reviewer: I think "Demands" should be just a plural, not a saxon genitive...

> _Demand_'s [Computational Payload Manifests](../requestor-tutorials/vm-runtime/computation-payload-manifest.md) can specify list of URLs which may get called by service running on a Provider. If Manifest declares requests to URLs which domains are not whitelisted it must come with a [signature and app author's public certificate](../requestor-tutorials/vm-runtime/computation-payload-manifest.md). By default domains `whitelist` is empty.

Reviewer: ```suggestion
The [Computational Payload Manifests](../requestor-tutorials/vm-runtime/computation-payload-manifest.md) embedded in the Demands can specify a list of URLs which may get called by the services running on a Provider. If the manifest declares requests to URLs in domains that are not whitelisted, it must come with a [signature and app author's public certificate](../requestor-tutorials/vm-runtime/computation-payload-manifest.md). By default, the domains `whitelist` consists of a curated set of public websites and APIs like github, dockerhub or public Ethereum nodes.
```
